### ISSUE TITLE
Suggest add a part in People/Member page for those who are "Big Person" and once attend AMC meetings.

### ISSUE THREAD

**Issue by newip** [BUPT/ai-ml.club#315]
With AMC development in fast progess, there are quite a few people who are very good in AI-ML domain have had attended one or more AMC events. To remember their presence in person, we may add a part of the content in the People/Member page to make their Bio present in club pages. 

If agreed, please add a file in repo and make update roles and rules. Tks.

**Comment by huan** [comment]
Good idea. We can discuss this topic in person at the next AMC meeting.
